symbian/symbianish.h

```diff
@@ -122,7 +122,7 @@
 #ifndef PERL_SYS_TERM_BODY
 #define PERL_SYS_TERM_BODY()	HINTS_REFCNT_TERM; OP_REFCNT_TERM;          \
 				PERLIO_TERM; MALLOC_TERM; CloseSTDLIB();    \
-                                PERL_LOCALE_TERM
+                                LOCALE_TERM; ENV_TERM;
```

Comment:
I suggest pulling the correction to PERL_LOCALE_TERM into a separate commit.

Reply:
Good catch.  I didn't even realize this was a bug fix

Reply:
Ah, that it is symbian explains why it didn't show up in testing.

Reply:
That separate commit is 76318cfdec9c5f0f70c97173e383b30601dd82dd